src/main/java/dp/handler/Handler.java

```diff
@@ -114,6 +114,7 @@
 
     @KafkaListener(topics = "${KAFKA_TOPIC:common-output-created}")
     public void listen(final ExportedFile message) {
+        info().traceID(null);
```

Comment:
Question: Why is this needed? I think the trace ID is provided by the request? Or is the same ID hanging around between each message?

Reply:
I think this might actually be a different use case for the logger.

The logger was built for HTTP requests - each request spawns a new thread and the values like trace ID are stored on the thread local map for the duration of the request. On completion the thread dies and the ID is removed. I'm wondering if this handler actually keeps the same thread alive to handle multiple messages? 

I'm trying to determine if this is an actual bug in the log library or if we're just using the logger in a different way.

Reply:
Sorry, I'm not too familiar with this repo and the logger and maybe there is a better way to implement the change.
The issue that needs to be fixed is that trace ID doesn't change from one kafka message to another.
I assumed that we handle a single message at a a time (I didn't test the case of handling concurrent messages, is it also possible?)

If the logger creates a new trace ID for each thread, maybe the proper implementation would be to make sure that each Kafka message is handled in a separate thread?

Reply:
I tried to create a thread for each new Kafka message, but the traceID is not automatically generated (or at least it doesn't appear if I do info().log(...))

Reply:
I think I know what you ment. In dp-logging, we have to extend the BaseEvent class with :
```
    public T beginKafkaConsumer() {...}
    public T endKafkaConsumer() {...}
    public T beginKafkaProducer() {...}
    public T endKafkaProducer() {...}
```